src/demo-app/baseline/baseline-demo.html

```diff
@@ -10,7 +10,7 @@
     | Text 3 |
     <md-radio-button value="option_3">Radio 3</md-radio-button>
     | Text 4 |
-    <md-input>Label</md-input>
+    <md-input-wrapper><input placeholder="Input"></md-input-wrapper>
```

Comment:
2 things;

1. since we're not remove the input completely (yet), could you leave 1 here and add the new wrapper?
2. could you insert new lines and tabulation? it might affect the baseline.